integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/Compactor.scala

```diff
@@ -69,11 +69,7 @@ object Compactor {
     // trigger event for compaction
     val operationContext = new OperationContext
     val alterTableCompactionPreEvent: AlterTableCompactionPreEvent =
-      AlterTableCompactionPreEvent(compactionCallableModel.sqlContext.sparkSession,
-        carbonTable,
-        carbonLoadModel,
-        mergedLoadName,
-        sc)
+      AlterTableCompactionPreEvent(carbonTable, carbonMergerMapping, mergedLoadName, sc)
```

Comment:
please modify line 55, it is not storePath, it is tablePath

Reply:
ok